frame/staking/README.md

```diff
@@ -90,9 +90,9 @@ valid behavior_ while _punishing any misbehavior or lack of availability_.
 
 Rewards must be claimed for each era before it gets too old by `$HISTORY_DEPTH` using the
```

Comment:
```suggestion
Rewards must be claimed for each era before it gets too old by [`HistoryDeth`] using the
```

This should create a correct link I think: https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html